secp256k1jni/natives/osx_64/libsecp256k1.la

```diff
@@ -17,7 +17,7 @@ old_library='libsecp256k1.a'
 inherited_linker_flags=' '
 
 # Libraries that this one depends upon.
-dependency_libs=''
+dependency_libs=' -L/usr/local/opt/gmp/lib -lgmp'
```

Comment:
Hmm.. this seems strange. My understanding is that libsecp256k1 has no external deps

Reply:
Fixed. For reference I ran configure with `--with-bignum=no` to get rid of this.